language/cloud-client/v1/snippets.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-# Copyright 2016 Google, Inc.
+# Copyright 2018 Google, LLC.
```

Comment:
For the future, I've been told we only need to update this for new files. 
It's just `LLC` no `.`